README.md

```diff
@@ -155,9 +155,31 @@ spark.sql("select count(*) from lineitem").show
 spark.sql("select ti_version()").show
 ```
 
-## TiDB Data Source API
+## Write Data To TiDB using TiDB Connector
+TiSpark natively supports writing data to TiKV using Spark Data Source API. ACID is also guaranteed.
+
+For example:
+
+```scala
```

Comment:
Please add comment in code that connection to TiDB is for lock instead of data write.

Reply:
fixed